tests/test_modeling_dpr.py

```diff
@@ -104,7 +104,8 @@ def prepare_config_and_inputs(self):
             token_labels = ids_tensor([self.batch_size, self.seq_length], self.num_labels)
             choice_labels = ids_tensor([self.batch_size], self.num_choices)
 
-        config = BertConfig(
```

Comment:
This is dangerous went new params are added to Bert, it'll lead to silent errors

Reply:
Nice catch